src/main/java/com/csc530/sat/type/DDType.java

```diff
@@ -0,0 +1,5 @@
+package com.csc530.sat.type;
+
+public interface DDType<T> {
```

Comment:
It's an abstraction for the Decision Diagram, and it needs a value cause that's basically all it is